tasks/eva_2156/deprecate_variant_from_contig.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+from argparse import ArgumentParser
+import pymongo
+
+from ebi_eva_common_pyutils.config_utils import get_mongo_uri_for_eva_profile
+from ebi_eva_common_pyutils.logger import logging_config as log_cfg
+
+logger = log_cfg.get_logger(__name__)
+
+
+def inactive_object(variant):
+    return {
+        "eventType": "UPDATED",
```

Comment:
We are deprecating the variants so using the "DEPRECATED" [event type](https://github.com/EBIvariation/accession-commons/blob/master/accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/core/models/EventType.java) is more accurate.